### PR TITLE
Fix stylelint issues

### LINKED ISSUE #503: Fix stylelint issues
# 🐛 Bug Report

This. Also, `yarn stylelint:watch`.

## How to reproduce

Steps to reproduce the issue:

 1. Open the application in a browser
 2.
 3.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Browser          |
| Operating System |
|                  |

## Related

 -

### PR DESCRIPTION
**Pull request by natkaratkova**
## Related issues and PRs
Fixes #503
 - 

## Description
Implements stylelint fixes

## Impacted Areas in the application
- PR https://github.com/neherlab/covid19_scenarios/pull/548 Using CSS `transition` for `width` and `max-width` is a performance issue. It requires to rewrite a lot of functionality so I disabled the line and added a comment.

- removed `stylelint-group-selectors`. It has some false positive scenarios. And anyway this optimization will be done via CSS modification tool.

- removed `stylelint-use-nesting` because of conflicts with scss nesting

### CODE REVIEW

File: src/components/Main/Mitigation/MitigationTable.tsx [hunk 1]
```diff
@@ -135,22 +135,20 @@ function MitigationIntervalComponent({
...
-              className={`form-control item-value ${valueError ? 'border-danger' : ''}`}
-              id={`containment.mitigationIntervals[${index}].mitigationValue`}
-              name={`containment.mitigationIntervals[${index}].mitigationValue`}
-              type="number"
-              step={1}
-              min={0}
-              max={100}
-            />
-          </div>
+          <MitigationDatePicker
```
Comment (gj262) [inline src/components/Main/Mitigation/MitigationTable.tsx:138]: Hi @natkaratkova, thanks for working on this and I apologize for this horrendous CSS that I created.
One difference between the current behavior and your PR is that the interval switches to 3 rows when the space for the mitigation table is < 580px. In the current implementation the date range and value are kept in a row. Is there a way to get that behavior in your PR?

Comment (natkaratkova) [reply src/components/Main/Mitigation/MitigationTable.tsx:138]: hi @gj262, no worries, your CSS is very reasonable :) Thanks for the feedback, I see. I'll fix this

Comment (natkaratkova) [reply src/components/Main/Mitigation/MitigationTable.tsx:138]: I continued without a row (to stay within stylelint specificity limits). Hope hasn't broken anything. The "very narrow" view has changed a little but it fixed some date picker overlap Screen Shot 2020-04-16 at 05 21 20

**Comment by ivan-aksamentov** [comment]
@natkaratkova Thanks for the contribution! We will merge right after we release 1.1 later today (hopefully).

If you have further suggestions regarding stylelint config (or any other config, or dev experience in general), or regarding the panel expansion transition in https://github.com/neherlab/covid19_scenarios/pull/548, don't hesitate to manifest them! :)

**Comment by ivan-aksamentov** [comment]
@natkaratkova Nataliya, I've merged your changes, thank you.

I had to replace the part in `MitigationTable`  with Nicks' new slider-based implementation.
Nick is a computational biologist, so that code may need some love from a frontend professional. If you are available, feel free to take a look.

**Pull request merged**